core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeExternalSorter.java

```diff
@@ -90,9 +123,18 @@ public UnsafeExternalSorter(
     this.prefixComparator = prefixComparator;
     this.initialSize = initialSize;
     // Use getSizeAsKb (not bytes) to maintain backwards compatibility for units
-    this.fileBufferSizeBytes = (int) conf.getSizeAsKb("spark.shuffle.file.buffer", "32k") * 1024;
-    this.pageSizeBytes = conf.getSizeAsBytes("spark.buffer.pageSize", "64m");
-    initializeForWriting();
+    // this.fileBufferSizeBytes = (int) conf.getSizeAsKb("spark.shuffle.file.buffer", "32k") * 1024;
```

Comment:
This configuration is threaded down to the DiskBlockObjectWriter to control its write buffer size.  It may not be super important to make this tuneable.